.rubocop_todo.yml

```diff
@@ -100,7 +100,6 @@ Layout/DotPosition:
     - 'app/models/cas_seeds/client_shelter_agency_contact.rb'
     - 'app/models/cas_seeds/opportunity_housing_subsidy_admin_contact.rb'
     - 'app/models/concerns/related_default_contacts.rb'
-    - 'app/models/contact.rb'
```

Comment:
Pretty sure this didn't get fixed.  Let's leave the todos here unless we fix them.